lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -56,6 +56,12 @@ class DiskImageFormat(Enum):
     RAW = "raw"
 
 
+# Possible provisioning tools
+class InitSystem(Enum):
```

Comment:
This type isn't a part of the runbook schema. So, it should to `context.py` instead.